themes/atvp/css/imports/breakpoints.css

```diff
@@ -2,3 +2,4 @@
 @custom-media --desktop-nav (min-width: 62.501em);
 @custom-media --small-viewport (max-width: 44em);
 @custom-media --larger-than-small-viewport (min-width: 44.01em);
+@custom-media --larger-than-max-width (min-width: 87.5em);
```

Comment:
Just curious about the advantages of using `em` as the unit for break point dimensions. For me, it's harder to get a general idea of how wide or narrow a break point is at a glance, but that might be something I'll eventually get used to.

That said, I could see the decimal differences between a min an max at a common width potentially leading to a bit of slop or ambiguity (i.e. `max-width: 44em` and `min-width: 44.01em`). Not the best example, but if one layout adjustment takes place at a min width and a different one at a max width, there are probably a couple pixels in between where things could get goofy. I'd probably also flip that pattern (i.e. `max-width: 43.99em` and `min-width: 44em`), but that's undoubtedly just personal preference :).